software-engineering-policies/SourceControl/SourceControlPolicy.md

```diff
@@ -9,34 +9,50 @@
 * Any source code found in any other source control method must first be migrated and proven to build via a CI pipeline.
```

Comment:
Do we need to mention code found outside of source control, that might not have a build process, i.e. standalone legacy python files (still in use), being moved into AzDo purely for source control?